Sources/Instructions/Managers/Internal/OverlayStyleManager/TranslucentOverlayStyleManager.swift

```diff
@@ -10,17 +10,20 @@ class TranslucentOverlayStyleManager: OverlayStyleManager {
     // MARK: Private Properties
     private var onGoingTransition = false
     private let color: UIColor
-
+    private var cutoutBorderColor: UIColor
+    
```

Comment:
Swiftlint complains about trailing whitespace here. :/

Note: it's happening on lines 132 & 136 as well.